CHANGELOG.md

```diff
@@ -1,3 +1,14 @@
+## 7.11.2 (May 30, 2023). Tested on Artifactory 7.55.14
+
+IMPROVEMENTS:
+
+* resource/scoped_token is migrated to Plugin Framework, improved attribute validation.
+
+  PR:     [#741](https://github.com/jfrog/terraform-provider-artifactory/pull/741)
+
+Note: Resources `ldap_group_setting` and `ldap_setting` won't work with Artifactory version > 7.57.1. 
```

Comment:
Let's follow [HashiCorp recommendation](https://developer.hashicorp.com/terraform/plugin/best-practices/versioning#changelog-specification) and use `NOTES` section. Or we can use our own `KNOWN ISSUES` section. This is currently not prominent enough.

Reply:
`NOTES` is ok.